test/test_util/test_aggressive_momentum_model_free_util.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+
+
+from fi_nomad.util.aggressive_momentum_model_free_util import (
+    increase_momentum_beta,
+    increase_momentum_upper_bound_beta_bar,
+    decrease_momentum_beta,
+    validate_hyperparameters,
+)
+
+from fi_nomad.types.kernelInputTypes import AggressiveMomentumAdditionalParameters
+
+
+PKG = "fi_nomad.util.aggressive_momentum_model_free_util"
+
+
+def test_increase_momentum_beta_regular() -> None:
+    beta = 0.5
+    gamma = 1.2
+    beta_bar = 1.0
+    expected_result = 0.6
```

Comment:
Likewise, I'd consider making the computation explicit for the expected result.

So like, `expected_result = beta * gamma` or whatever.